hydroshare/local_settings.template

```diff
@@ -105,7 +105,7 @@ IRODS_AUTH = 'wwwHydroProxy'
 IRODS_GLOBAL_SESSION = True
 
 # Remote user zone iRODS configuration
-REMOTE_USE_IRODS = False
+REMOTE_USE_IRODS = True
```

Comment:
This definitely shouldn't have been committed

Reply:
How does the local_settings.template work, anyway? I always wondered if it was vestigial or if CUAHSI/RENCI Jenkins or other scripts/manual deployment steps use it....